src/policy/policy.h

```diff
@@ -19,8 +19,6 @@ class CCoinsViewCache;
 class CFeeRate;
 class CScript;
 
-/** Default for -blockmaxweight, which controls the range of block weights the mining code will create **/
-static constexpr unsigned int DEFAULT_BLOCK_MAX_WEIGHT{MAX_BLOCK_WEIGHT - 4000};
```

Comment:
This should remain logically separate from consensus constant.

Reply:
Seems reasonable to keep and just make it `DEFAULT_BLOCK_MAX_WEIGHT{MAX_BLOCK_WEIGHT};`

Reply:
Done.